docs/src/index.md

```diff
@@ -1,12 +1,68 @@
 # Chmy.jl
 
-Documentation for [Chmy.jl](https://github.com/PTsolvers/Chmy.jl).
+[Chmy.jl](https://github.com/PTsolvers/Chmy.jl) is a backend-agnostic toolkit for finite difference computations on multi-dimensional computational staggered grids. Chmy.jl features task-based distributed memory parallelisation capabilities.
 
-Information about the GPU4GEO project can be found on the [GPU4GEO website](https://ptsolvers.github.io/GPU4GEO/).
+## Installation
 
-## Doc content
+To install Chmy.jl, one can simply add it using the Julia package manager:
 
-- [Concepts](concepts/overview.md)
-- [Examples](examples/overview.md)
-- [Usage information](usage/runtests.md)
-- [API reference](lib/modules.md)
\ No newline at end of file
+```julia
+using Pkg
+Pkg.add("Chmy")
```

Comment:
Maybe one could add a note that until it is registered, one should add it via the URL?

Reply:
I think it depends on when the registration would be, if it's in near future (when the doc is up, the registry is up or so). I would keep it as it is.

Reply:
I guess having the guide on how to install from the specific branch would be nice nevertheless, added it in the [commit](https://github.com/PTsolvers/Chmy.jl/pull/24/commits/9192a5f2d4dbb2cab543d64a23f297670cbafccf) here.